docs/Developer's Guide.md

```diff
@@ -86,6 +86,12 @@ Sometimes, you would like to connect the frontend on your local computer to the
 1. Your local machine must be configured as an SSH host
 
    [Windows Installation](https://learn.microsoft.com/en-us/windows-server/administration/openssh/openssh_install_firstuse?tabs=gui).
+   
+   - If you are on Windows 11+, you may need to update the firewall to allow inbound connections on a public network.
+      - Search for `Check Firewall Status` in the start menu
+      - Click `Turn Windows Defender Firewall on or off`
+      - Uncheck `Block all incoming connections, including...` under the `Public network settings` header
+      - Click Ok
```

Comment:
```suggestion
      - Click Ok
   - Also on Windows, if you are a local administrator, you will not be able to use SSH key authentication unless you add the public SSH key to `C:\Program Data\ssh\administrators_authorized_keys` rather than to `C:\Users\[your username]\.ssh\authorized_keys`. Create the file if it does not exist.
```